### ISSUE TITLE
Games through Proton take Ages to launch

### ISSUE THREAD

**Issue by MartinKurtz** [ValveSoftware/steam-for-linux#11331]
#### Your system information

* Steam client version (build number or date): 1726604483
* Distribution (e.g. Ubuntu): Debian 12
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes, System is freshly updated and issue still occurs
* Steam Logs: [generate by running this command in a terminal `tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs`] i have the logs compressed, will upload in a sec
* GPU: AMD

#### Please describe your issue in as much detail as possible:
Basically a game launched takes ages to launch. Some research has concluded that this could be related to steam cloud savegames, but i cant be sure. Proton games launched without steam through lutris are unaffected, even if the same version of proton is used

#### Steps for reproducing this issue:

1. Install to my knowledge any game
2. Enable Proton Proton 9, no beta or experimental
3. Press Launch
4. Wait 15 Minutes or so
5. Game launches eventually

**Comment by MartinKurtz** [comment]
and here we have the link to the logs


https://drive.proton.me/urls/6TCA34MMRM#Ly7bh6rspgNh
